yt_dlp/extractor/netverse.py

```diff
@@ -0,0 +1,175 @@
+            'display_id': 'tetangga-baru',
+            'duration': 1406,
+        },
+    }, {
+        # /video url
+        'url': 'https://www.netverse.id/video/pg067482-hellojadoo-season1',
+        'title': 'Namaku Choi Jadoo',
+        'info_dict': {
+            'id': 'x887jzz',
+            'ext': 'mp4',
+            'thumbnail': 'https://storage.googleapis.com/netprime-live/images/webseries/thumbnails/2021/11/619cf63f105d3.jpeg',
+            'season': 'Season 1',
+            'episode_number': 1,
+            'description': 'md5:c616e8e59d3edf2d3d506e3736120d99',
+            'title': 'Namaku Choi Jadoo',
+            'series': 'Hello Jadoo',
+            'episode': 'Episode 1',
+            'age_limit': 0,
+            'like_count': int,
+            'view_count': int,
+            'tags': ['PG067482', 'PG067482-HelloJadoo-season1'],
+            'duration': 780,
+            'display_id': 'pg067482-hellojadoo-season1',
+            'uploader_id': 'x2ir3vq',
+            'uploader': 'Net Prime',
+            'timestamp': 1645764984,
+            'upload_date': '20220225',
+        },
+        'skip': 'This video get Geo-blocked for some country'
+    }]
+
+    def _real_extract(self, url):
+        display_id, program_json = self._call_api(url)
+        videos = program_json['response']['videos']
+
+        return {
+            '_type': 'url_transparent',
+            'ie_key': DailymotionIE.ie_key(),
+            'url': smuggle_url(videos['dailymotion_url'], {'query': {'embedder': 'https://www.netverse.id'}}),
+            'display_id': display_id,
+            'title': videos.get('title'),
+            'season': videos.get('season_name'),
+            'thumbnail': traverse_obj(videos, ('program_detail', 'thumbnail_image')),
+            'description': traverse_obj(videos, ('program_detail', 'description')),
+            'episode_number': videos.get('episode_order'),
+            'series': traverse_obj(videos, ('program_detail', 'title')),
+        }
+
+
+class NetversePlaylistIE(NetverseBaseIE):
+    _VALID_URL = r'https?://(?:\w+\.)?netverse\.id/(?P<type>webseries)/(?P<display_id>[^/?#&]+)'
+    _TEST = {
+        'url': 'https://netverse.id/webseries/tetangga-masa-gitu',
+        'info_dict': {
+            'id': 'tetangga-masa-gitu',
+            'title': 'Tetangga Masa Gitu',
+        },
+        'playlist_count': 46,
+    }
+
+    def parse_playlist(self, url, page_num):
+        _, playlist_json = self._call_api(url, query={'page': page_num})
+        for slug in traverse_obj(playlist_json, ('response', 'related', 'data', ..., 'slug')):
+            yield self.url_result(f'https://www.netverse.id/video/{slug}', NetverseIE)
+
+    def _real_extract(self, url):
+        _, playlist_data = self._call_api(url)
+
+        # TODO: get video from other season
+        # The season has id and the next season video is located at api_url/<season_id>?page=<page>
+        # still not not sure about number in OnDemandPagedList
```

Comment:
will you be addressing this todo now? or should I merge without it?

```suggestion
        # FIXME: get video from other season from api_url/<season_id>?page=<page>
```